clients/src/main/resources/common/message/BeginQuorumEpochRequest.json

```diff
@@ -23,32 +23,33 @@
   "flexibleVersions": "1+",
   "fields": [
     { "name": "ClusterId", "type": "string", "versions": "0+",
-      "nullableVersions": "0+", "default": "null"},
+      "nullableVersions": "0+", "default": "null",
+      "about":  "The cluster id."},
     { "name": "VoterId", "type": "int32", "versions": "1+", "ignorable": true, "default": "-1", "entityType": "brokerId",
-      "about": "The replica id of the voter receiving the request" },
-    { "name": "Topics", "type": "[]TopicData",
-      "versions": "0+", "fields": [
+      "about": "The replica id of the voter receiving the request." },
+    { "name": "Topics", "type": "[]TopicData", "versions": "0+",
+      "about": "The topics to begin the epoch on.", "fields": [
```

Comment:
KIP-595 doesn't really give the terminology to use here. To me, "to begin the epoch on" sounds a bit strange. I suggest "The topics to begin the epoch".